ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -289,28 +291,22 @@ UADeepLinkDelegate, UAPushNotificationDelegate {
 
     private func editAttributes(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
```

Comment:
You might rename private methods to `editChannelAttributes` throughout, to be symmetric with `editNamedUserAttributes`. Probably not worth a change to the public I/F, but maybe even there.